js/app/Visualization/UI/Search.js

```diff
@@ -34,6 +34,12 @@ define([
           '</div>'
       });
 
+      $(self.dialog.containerNode).find(".query").keyup(function(event) {
+        if (event.keyCode == 13) {
```

Comment:
@redhog From jquery documentation (https://api.jquery.com/event.which/)
`The event.which property normalizes event.keyCode and event.charCode. It is recommended to watch event.which for keyboard key input.`